certif/app/styles/pages/authenticated/sessions/import.scss

```
@@ -124,6 +124,10 @@
     display: grid;
     grid-template-columns: 1fr 10fr 2fr;
     padding: 24px 16px 24px 0;
+
+    > label, button {
+      width: 164px;
```

Même commentaire qu'au dessus 😄 

```suggestion
      min-width: 164px;
```

---

vu ensemble, on a pu appliquer le min-width pour l'ajout de candidat mais pas pour l'import en masse à cause de l'usage du grid.

Proposition : Créer un composant commun pour les deux pages